src/theme/themes/pfr.scss

```diff
@@ -48,7 +48,7 @@
       // radio-button-font-color: var(--ion-color-primary),
       ion-item-background: var(--ion-color-gray-light),
       // task-progress-bar-color: var(--ion-color-primary),
-      // checkbox-background-color: white,,,,
+      // checkbox-background-color: white,
```

Comment:
I'm not sure why this change is included here, as it was already committed as part of #2299. I've tried changing the base of this PR away from master and back again, but this supposed difference remains. (That method did get rid of some erroneous differences that were part of #2294, which I merged into this feature branch before it was merged to master).